mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowHttpException.java

```diff
@@ -0,0 +1,51 @@
+package org.mlflow.tracking;
+
+/**
+ * HTTP exception.
+ */
+public class MlflowHttpException extends MlflowClientException {
+
+  public MlflowHttpException(int statusCode, String reasonPhrase) {
+    super("statusCode=" + statusCode + " reasonPhrase=" + reasonPhrase);
+    this.statusCode = statusCode;
+    this.reasonPhrase = reasonPhrase;
+  }
+
+  public MlflowHttpException(int statusCode, String reasonPhrase, String bodyMessage) {
+    super("statusCode=" + statusCode + " reasonPhrase=[" + reasonPhrase + "] bodyMessage=["
+      + bodyMessage + "]");
```

Comment:
Seems like this string representation is different from the one in the previous constructor because it has `[` and `]` around reasonPhrase.

Reply:
Updated above.